instrumentation/servlet/servlet-common/library/src/main/java/io/opentelemetry/instrumentation/servlet/ServletNetAttributesExtractor.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.servlet;
+
+import io.opentelemetry.instrumentation.api.instrumenter.net.NetAttributesExtractor;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+public class ServletNetAttributesExtractor<REQUEST, RESPONSE>
+    extends NetAttributesExtractor<
+        ServletRequestContext<REQUEST>, ServletResponseContext<RESPONSE>> {
+  private final ServletAccessor<REQUEST, RESPONSE> accessor;
+
+  public ServletNetAttributesExtractor(ServletAccessor<REQUEST, RESPONSE> accessor) {
+    this.accessor = accessor;
+  }
+
+  @Override
+  public @Nullable String transport(ServletRequestContext<REQUEST> requestContext) {
+    // return SemanticAttributes.NetTransportValues.IP_TCP;
+    return null;
+  }
+
+  @Override
+  public @Nullable String peerName(
+      ServletRequestContext<REQUEST> requestContext,
+      @Nullable ServletResponseContext<RESPONSE> responseContext) {
+    // return accessor.getRequestRemoteHost(requestContext.request());
```

Comment:
Because I realized that I need to avoid adding new attributes in this pr to avoid changing tests which would further increase the size of the pr. As I had already figured out how to fill some of the missing attributes I left them commented out so it would be easier to enable them in a subsequent pr.